merlin/models/tf/models/ranking.py

```diff
@@ -7,9 +7,58 @@
 from ..blocks.inputs import InputBlock
 from ..blocks.mlp import MLPBlock
 from ..core import Block, Model, ParallelPredictionBlock, PredictionTask
+from ..features.embedding import EmbeddingOptions
 from .utils import _parse_prediction_tasks
 
 
+def MLPModel(
+    schema: Schema,
+    embedding_dim: int,
```

Comment:
What’s the rational behind only exposing `embedding_dim` and not the rest of the options of `EmbeddingOptions `?

Reply:
Ah so to be clear, the proposal I posted in the comment below was meant to be a replacement this of the PR. So if we agree with the proposal, we can close this PR.

Reply:
We can close ok.